predict_on_raw_img.py

```diff
@@ -0,0 +1,40 @@
+"""Class for init the model, preprocess the image and make a prediction."""
+
+from transforms.acne_transforms import AcneTransformsTorch
+import torch
+from model.resnet50 import resnet50
+
+
+class ModelInit:
+    """Class that initialize the model and make prediction on single raw image."""
+
+    def __init__(self, model_type="model_ld_smoothing", path_checkpoint=None, device="cpu"):
+        """Init of the object."""
+        self.model_type = model_type
+        # Create model
+        num_acne_cls = 13 if model_type == "model_ld_smoothing" else 4
+        self.model = resnet50(num_acne_cls=num_acne_cls)
+        # load checkpoint
+        checkpoint = torch.load(path_checkpoint, map_location=torch.device(device))
+        self.model.load_state_dict(checkpoint["model_state_dict"])
+        # transforms
+        self.transform = AcneTransformsTorch(train=False)
+
+    def predict_on_raw_img(self, img):
```

Comment:
Not sure why this is called `raw` image. Images in ACNE04 are also "raw", and transforms that are used to preprocess are the same. I'd just call it `predict_on_image`